lib/src/grpc_api_impl/logging_impl.dart

```diff
@@ -371,17 +371,21 @@ class SharedLoggingService {
       ..partialSuccess =
           false /* for now we want to get notified if something goes wrong */;
     _entries.clear();
-    _clientStub.writeLogEntries(request).catchError((error, stack) {
-      // In case the logging API failed, we'll write the error message to
-      // stderr.  The logging daemon on the VM will make another attempt at
-      // uploading stderr via the logging API.
-      stderr.writeln('An error occured while writing log entries:\n'
-          'Error:$error\n'
-          '$stack');
-    }).whenComplete(() {
-      _outstandingRequests--;
-      _maybeClose();
-    });
+    unawaited(() async {
```

Comment:
I think this is the right way. We could also consider:
```
_clientStub.writeLogEntries(request).then((_) {}).catchError(....
```
As a way of making the types match.

Reply:
Yes, that could also have worked. but `try/catch` with `async/await` is just all around more sane :D